app/src/lib/stores/app-store.ts

```diff
@@ -407,6 +407,9 @@ const lastThankYouKey = 'version-and-users-of-last-thank-you'
 const pullRequestSuggestedNextActionKey =
   'pull-request-suggested-next-action-key'
 
+const underlineLinksKey = 'underline-links'
+const underlineLinksDefault = false
```

Comment:
```suggestion
const underlineLinksDefault = true
```
We want the default to be true but down in that if statement, set it to false if undefined. That way on next release, anyone who get its it will get it as false. Then, down the line we can remove that if statement, and all new users will get true.